blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+title: Bring GitOps to your private Kubernetes clusters with ArgoCD and inlets PRO
+description: Learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation and inlets PRO
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud multi-cloud gitops argocd
+author_img: jsiebens
+image: /images/2021-05-18-argocd-private-clusters/background.jpg
+date: 2021-05-18
+---
+
+In this post, you'll learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation.
+
+## GitOps with Argo CD
```

Comment:
I feel like GitOps with ArgoCD isn't the right heading here. The introduction should *define the problem* and *the benefit of a solution*, the technology should come second.

ArgoCD is sylised: "ArgoCD"

Reply:
I'm not sure about "ArgoCD". When you have a look at their website and documentation, they always write "Argo CD"

Reply:
OK you are probably right then.